libsolidity/lsp/LanguageServer.cpp

```diff
@@ -109,7 +109,7 @@ Json semanticTokensLegend()
 	tokenTypes.emplace_back("operator");
 	tokenTypes.emplace_back("parameter");
 	tokenTypes.emplace_back("property");
-	tokenTypes.emplace_back("std::string");
+	tokenTypes.emplace_back("string");
```

Comment:
This seems to be a false-positive that slipped through in our `using namespace std` purge: #14493.

Looking through that PR I see that it also broke indentation here: https://github.com/ethereum/solidity/blob/f344dc164e9d720f604ac5b345df1243587d6942/libsolidity/parsing/Parser.cpp#L2263-L2265

Could you fix that as well?